install-script.sh

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/env bash
+
+PWD=$(pwd)
+
+# first, backup any existing scripts if they are not symlinks
+if [[ -d "$HOME/.vim" && ! -L "$HOME/.vim" ]]; then
+  echo -e "moving ~/.vim to ~/.vim_backup"
+  mv ~/.vim ~/.vim_backup
+else
+  echo "$HOME/.vim has already been moved, is a symlink, or does not exist"
```

Comment:
That could be nice. It might make the script more complicated. I like doing piece by piece because the installation never fails, it just has various states of being finished.